v21.2/set-vars.md

```diff
@@ -93,68 +117,166 @@ The following demonstrates how to assign a list of values:
 
 {% include copy-clipboard.html %}
 ~~~ sql
-> SET search_path = pg_catalog,public;
+SET search_path = pg_catalog,public;
 ~~~
 
 {% include copy-clipboard.html %}
 ~~~ sql
-> SHOW search_path;
+SHOW search_path;
 ~~~
 
 ~~~
      search_path
-+--------------------+
+----------------------
   pg_catalog, public
 (1 row)
 ~~~
 
 ### Reset a variable to its default value
 
-{{site.data.alerts.callout_success}}You can use <a href="reset-vars.html"><code>RESET</code></a> to reset a session variable as well.{{site.data.alerts.end}}
+{{site.data.alerts.callout_success}}
+You can use <a href="reset-vars.html"><code>RESET</code></a> to reset a session variable as well.
+{{site.data.alerts.end}}
+
+{% include copy-clipboard.html %}
```

Comment:
Shouldn't all of these `include copy-clipboard.html` be changed to `include_cached copy-clipboard.html`?